reana_workflow_engine_yadage/utils.py

```diff
@@ -1,24 +0,0 @@
-# -*- coding: utf-8 -*-
-#
-# This file is part of REANA.
-# Copyright (C) 2018 CERN.
-#
-# REANA is free software; you can redistribute it and/or modify it
-# under the terms of the MIT License; see LICENSE file for more details.
-"""REANA-Workflow-Engine-yadage utility functions."""
```

Comment:
"utils" was there I presume because of analogy with Serial... But the other workflow engines don't use it, so we can definitely refactor...